ivy/container/utility.py

```diff
@@ -22,6 +21,20 @@ def static_all(
         *,
         out: Optional[ivy.Container] = None,
     ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.all. This method simply wraps the
+        function, and so the docstring for ivy.all also applies to this method
+        with minimal changes.
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([0, 1, 2]))
```

Comment:
all `ivy.Container` examples should have at least two leaves, so for example `x = ivy.Container(a=ivy.array([0, 1, 2]), b=ivy.array([3, 4, 5]))`